core/ecschema-metadata/src/test/UnitConversion/Convert.test.ts

```diff
@@ -17,9 +16,19 @@ interface TestData {
   expect: number;
 }
 
+/**
+ * Checks if two numbers are approximately equal within a given tolerance.
+ * @param a - The first number to compare.
+ * @param b - The second number to compare.
+ * @param epsilon - The tolerance within which the numbers are considered equal.
+ * @returns True if the numbers are approximately equal, false otherwise.
+ */
+function almostEqual(a: number, b: number, epsilon: number = 1.19209290e-7): boolean {
```

Comment:
do we need to copy this into the test code? why cant we export from UnitConversion and import from there for the tests?

Reply:
I notice the different epsilon values, but still think we can import the utility fn from the UnitConversion module and actually pass in the tolerance

Reply:
no i was against the idea of adding a static method, we need to be more vigilant with not immediately going to statics in JS world. its fine to export it from the UnitConversion.ts module itself, it wont be apart of our public api unless you explicitly export it in the barrel file & the fn is marked as public. if api-extractor complains about it not having a release tag, just mark it as internal.

Reply:
We are going to need a similar method in the quantity package for inverting units, which this package depends on. So I'd rather put it there and include it here. It's also similar to what native did. https://github.com/iTwin/imodel-native/blob/main/iModelCore/Units/src/Quantity.cpp#L112

Reply:
Hey @aruniverse I've updated the code according to the latest discussions, can you have a quick look? Thanks